internal/paging/paging.go

```diff
@@ -0,0 +1,23 @@
+package paging
+
+import (
+	"github.com/UpCloudLtd/upcloud-go-api/v8/upcloud/request"
+	"github.com/spf13/pflag"
+)
+
+type PageParameters struct {
+	size   int
+	number int
+}
+
+func (pp *PageParameters) ConfigureFlags(fs *pflag.FlagSet) {
+	fs.IntVar(&pp.size, "size", 100, "Number of entries to receive at most.")
+	fs.IntVar(&pp.number, "page", 0, "Page number to calculate first item to receive. Page numbers start from `1`.")
```

Comment:
These could also be `--page-size` and `--page-number` to be more descriptive, but those are quite verbose 🤔

Reply:
As they naturally are available only on `list` commands, these shorter ones seem fine to me.

Reply:
I was also thinking between `--size` and `--limit`. Go SDK uses size, but there that is used as field of `Page` struct. Maybe limit would clearer without that context 🤔

Reply:
Changed `--size` → `--limit`